lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -2273,6 +2282,38 @@ def wait_for_vm_ip(self, vm, poll=100, sleep=5):
 
         return facts
 
+    def get_vm_events(self, eventTypeIdList):
```

Comment:
Shouldn't this move to the module utils?

Reply:
Don't know, should it? Thought this is fine here, because also `wait_for_task` is here...
Do you mean the `ansible.module_utils.vmware` module?
Can you please elaborate more on the split?
From my point of view there is no real pattern for what stays in the module and what in utils.

Reply:
Typically, anything that can be reused by other modules, should be in the `anisble.module_utils.vmware` module. For now, this is the only module using it, so it can stay here.